packages/sources/css/src/components/text-input/src/index.css

```diff
@@ -132,14 +131,3 @@ textarea.vtmn-text-input {
   color: var(--vtmn-semantic-color_content-primary);
   fill: var(--vtmn-semantic-color_content-primary);
 }
-
-textarea.vtmn-text-input + .vtmn-text-input_counter {
-  color: var(--vtmn-semantic-color_content-secondary);
-  font-size: rem(14px);
-  line-height: 1;
-}
-
-.vtmn-text-input:not(:disabled):focus-visible + .vtmn-text-input_counter,
-.vtmn-text-input--error:not(:disabled) + .vtmn-text-input_counter {
-  color: var(--vtmn-semantic-color_content-primary);
-}
```

Comment:
are you sure??!

Reply:
Yes, the counter part of the text input is not official in design. I did it in advance but is unused and not reviewed today